content/courses/Ada_For_The_C_Embedded_Developer/chapters/02_Perspective.rst

```diff
@@ -2644,9 +2646,10 @@ Overloading
 ~~~~~~~~~~~
 
 In C, function names must be unique. Ada allows overloading, in which multiple
-subprograms can share the same name as long as the subprogram signatures (subprogram name, parameter
-types, and return types) are different. The compiler will be able to resolve
-the calls to the proper routines or it will reject the calls. For example:
+subprograms can share the same name as long as the subprogram signatures
+(number of parameters, the parameter types, and function return types) are
```

Comment:
I was wrong to include the number of parameters. It is part of the signature, but can be the same. In other words, that number can be used to disambiguate calls that are otherwise ambiguous, but need not be different for the overloading itself to be legal.